bindings/go/scip/parse_test.go

```diff
@@ -102,15 +106,22 @@ func checkRoundtrip(t *testing.T, index *Index) {
 func parseStreaming(t *testing.T, reader io.Reader) *Index {
 	parsedIndex := Index{}
 
-	indexVisitor := IndexVisitor{func(metadata *Metadata) {
-		parsedIndex.Metadata = metadata
-	}, func(document *Document) {
-		parsedIndex.Documents = append(parsedIndex.Documents, document)
-	}, func(extSym *SymbolInformation) {
-		parsedIndex.ExternalSymbols = append(parsedIndex.ExternalSymbols, extSym)
-	}}
+	indexVisitor := IndexVisitor{
+		VisitMetadata: func(ctx context.Context, metadata *Metadata) error {
```

Comment:
Let's use `_` if the parameter is not used.

In CHANGELOG.md, can you also add a Breaking change in the Unreleased section for Go bindings ?

Reply:
Done!